Source/ThirdParty/ANGLE/CMakeLists.txt

```diff
@@ -77,14 +77,19 @@ set(ANGLE_SOURCES
     ${angle_translator_essl_sources}
     ${angle_translator_glsl_and_vulkan_base_sources}
     ${angle_translator_glsl_sources}
-    ${angle_translator_glsl_symbol_table_sources}
     ${angle_translator_sources}
```

Comment:
I think it's in fact correct to use the essl symbol table unilaterally.
This is the frontend symbol table. The ANGLE frontend for WebGL is always GL ES.
(it shouldn't be a problem using the glsl, it's just overhead.)

Reply:
I went with this fix to cause as little disruption as possible, but that's a good point so I have filed [bug #245126](https://bugs.webkit.org/show_bug.cgi?id=245126) to handle that change later on.